### DISCUSSION TITLE
Where is the wiki entry for Audit log ?

### DISCUSSION THREAD

**Discussion by TechAspirant** [serenity-is/Serenity#5734] [Help] [answered]
Where can I find the wiki entry for code to enable audit log ?

**Comment by VictorTomaili** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/2023440/119264402-b4cd6380-bbeb-11eb-9d7a-8e6d91533d36.png)

**Reply by TechAspirant** [reply]
Thanks.
